packages/vite/src/node/server/middlewares/transform.ts

```diff
@@ -40,6 +40,16 @@ const debugCache = createDebugger('vite:cache')
 
 const knownIgnoreList = new Set(['/', '/favicon.ico'])
 
+const documentFetchDests = new Set<string | undefined>([
```

Comment:
Can the `string | undefined` be removed? Looks like the elements are all strings only. I suppose this affects the types for `documentFetchDests.has` below.

Reply:
Ah, I see. I'll change it.